docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Create shoot clusters in Azure
+level: beginner
+category: Getting Started
+scope: app-developer
+---
+
+# Create shoot clusters in Azure
+
+### Prerequisites
+
+-   You need an Azure account.
```

Comment:
You need to have as well as Azure Service Principal assigned to your subscription or the permissions to create one in the Active Directory tenant and assign it to your subscription.